프로그래머스/1/12982. 예산/예산.js

```diff
@@ -0,0 +1,10 @@
+function solution(d, budget) {
+  let answer = 0;
+  d.sort((a, b) => a - b);
+  for (let i = 0; i < d.length; i++) {
+    if (d[i] > budget) break;
+    budget -= d[i];
+    answer++;
+  }
```

Comment:
```suggestion
let answer = d.reduce((acc, cur) => {
    if (budget >= cur) {
      budget -= cur; 
      acc++; 
    }
    return acc;
  }, 0);
```
 reduce 함수의 초기값으로 0을 설정하여 answer 변수의 초기화와 최종 값 계산을 동시에 수행할수 있도록 하도 좋을것 같다는 생각이 들어요 !!